### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by irgalamarr**
Since we've discussed that all requests for new features or improvements should ideally be made through discussions, and issues should be brought to discord, I've made only a template for bugs. 

In other cases, when you try to create an issue for a feature or question, the corresponding page will open instead of the issue creation page.

By placing these configs in this repo, this config will apply to all repositories in the organization,

### CODE REVIEW

**Comment by irgalamarr** [comment]
Screenshot 19 02 2024 um 17 23 56 PM

**Comment by irgalamarr** [comment]
Screenshot 19 02 2024 um 17 03 26 PM
Screenshot 19 02 2024 um 17 03 35 PM

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name:  🐞 Bug Report
+description: Create a report to help us improve
+labels: ["bug", "new issue"]
```
Comment (psanders) [inline .github/ISSUE_TEMPLATE/bug.yml:3]: I noticed we're missing a 'new issue' label for both the client and server repos. Any plans to add that label soon?

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋🎉 Thanks for taking the time to fill out this bug report.
+
+        Before you post:
+        - Please search for existing issues to avoid creating duplicate requests.
+        - If you only have a question, you may get a faster response by asking in [our Discord server](https://).
```
Comment (psanders) [inline .github/ISSUE_TEMPLATE/bug.yml:12]: We are missing the link the server in here.

**Review by psanders — CHANGES_REQUESTED** [review]
Added two small notes, but otherwise, I love everything about this template.

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (irgalamarr) [reply .github/ISSUE_TEMPLATE/bug.yml]: changed to `unconfirmed`

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+description: Create a report to help us improve
+labels: ["bug", "new issue"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋🎉 Thanks for taking the time to fill out this bug report.
+
+        Before you post:
+        - Please search for existing issues to avoid creating duplicate requests.
```
[the remark this answers is not in the document]
Comment (irgalamarr) [reply .github/ISSUE_TEMPLATE/bug.yml]: added a required checkbox for dublicates

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💡 Feature Request
+    url: https://github.com/orgs/peers/discussions
```
[the remark this answers is not in the document]
Comment (irgalamarr) [reply .github/ISSUE_TEMPLATE/config.yml]: updated, thanks 😊

**Review by psanders — APPROVED** [review]

**Review by psanders — APPROVED** [review]

**Pull request merged**